DataFormats/Streamer/src/classes_def.xml

```diff
@@ -1,6 +1,8 @@
 
 <lcgdict>
- <class name="edm::StreamedProduct" ClassVersion="12">
+ <class name="edm::StreamedProduct" ClassVersion="14">
+  <version ClassVersion="14" checksum="855998814"/>
+  <version ClassVersion="13" checksum="87489871"/>
```

Comment:
There should be no reason for you to mave to change this. The problem which caused this should be fixed by Core software in a seperate build. Does rebasing make the problem go away?